target/linux/ipq40xx/image/Makefile

```diff
@@ -183,6 +183,21 @@ define Device/engenius_eap1300
 endef
 TARGET_DEVICES += engenius_eap1300
 
+define Device/engenius_eap2200
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	BOARD_NAME := eap2200
+	DEVICE_DTS := qcom-ipq4019-eap2200
+	DEVICE_TITLE := EnGenius EAP2200
```

Comment:
Use recently introduced DEVICE_VENDOR and DEVICE_MODEL instead of DEVICE_TITLE (e.g. have a look at the subsequent device in the Makefile).

Reply:
Thanks for advice.